frontend/src/components/common/Header/Header/Header.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import BurgerMenu from '../BurgerMenu/BurgerMenu';
+import styles from './Header.module.scss';
+import Button from '../../../ui-kit/components/Button';
+import { ICONS } from '../../../ui-kit/icons';
+import InputSearch from '../../../ui-kit/components/Input/InputSearch';
+import ROUTES from '../../../constants/routers';
+
+const Header = () => {
+  const [sticky, setSticky] = useState(false);
+
+  window.onscroll = () => {
+    if (window.scrollY > 70) {
+      setSticky(true);
+    } else {
+      setSticky(false);
+    }
+  };
+  return (
+    <div>
+      <BurgerMenu />
+      <header>
+        <div className={styles.containerTop}>
+          {' '}
+          <NavLink to={ROUTES.home}>
+            <img
+              className={styles.logo}
+              src="images/logo.svg"
+              alt="логотип магазину 'Квітково'"
+            />
+          </NavLink>
+          <div className={styles.contacts}>
+            <div className={styles.location}>
+              <Button
+                variant="no-border"
+                label="Київ"
+                padding="padding-header-even"
+                reverse="true"
+                icon={<ICONS.location />}
+              />
+            </div>
+
+            <a className={styles.phoneLink} href="tel:+380937777777">
+              <Button
+                variant="no-border"
+                label="(093) 777-77-77"
+                padding="padding-header-even"
+                reverse="true"
+                icon={<ICONS.phone />}
+              />
+            </a>
+          </div>
+          <ul>
+            <li>
+              <NavLink
```

Comment:
Make it an array of objects and output it through map, it will be much less code